Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -376,4 +376,100 @@ codeunit 4022 "GP Vendor Migrator"
         HelperFunctions.UpdateFieldValue(RecordVariant, GPVendorTransactions.FieldNo(PYMTRMID), JToken.AsObject(), 'PYMTRMID');
         HelperFunctions.UpdateFieldWithValue(RecordVariant, GPVendorTransactions.FieldNo(GLDocNo), DocumentNo);
     end;
+
+    procedure MigrateVendorEFTBankAccounts()
+    var
+        GPSY06000Table: Record GPSY06000;
+        VendorTable: Record Vendor;
+        VendorBankAccountTable: Record "Vendor Bank Account";
+        GeneralLedgerSetupTable: Record "General Ledger Setup";
+        VendorBankAccountExists: Boolean;
+        CurrencyCode: Code[10];
+    begin
+        if not GPSY06000Table.FindSet() then
+            exit;
+
+        repeat
+            if not GPSY06000Table.INACTIVE and VendorTable.Get(GPSY06000Table.CustomerVendor_ID) then begin
```

Comment:
It is better to split the IF statement to be like
IF not GPSY06000Table.INACTIVE then
   if VendorTable.Get(GPSY06000Table.CustomerVendor_ID) then
   
This way we can save one GET statement per GPSY06000Table record.  AL will always run all conditions in IF statement (it is bad, but it has always been like this).

Alternative is to set the key on the GPSY06000 on INACTIVE and to do GPSY06000Table.SetRange(Inactive, false) before findset.

To avoid the get on the vendor table you can add the flow field to the GPSY06000 table named Vendor Exists that is of type lookup on the vendor No. Ping me if you need help with defining the flow field. Then you can eliminate the both conditions with 
GPSY06000Table.SetRange(Inactive, false);
GPSY06000Table.SetRange("Vendor Exists", true);
if not GPSY06000Table.FindSet() then
exit;